src/apiCalls.tsx

```diff
@@ -0,0 +1,13 @@
+const fetchRandomDrinks = () => {
+  return fetch("https://www.thecocktaildb.com/api/json/v1/1/random.php").then(
+    (response) => response.json()
+  );
+};
+
+const fetchAllCocktails = () => {
+  return fetch(
+    "https://www.thecocktaildb.com/api/json/v1/1/filter.php?g=Cocktail_glass"
+  ).then((response) => response.json());
+};
```

Comment:
Aside from the URL here, these fetch calls look really similar. Can you think of any way to DRY these up?